.travis.yml

```diff
@@ -20,7 +20,7 @@ language: python
 # Default Python version is usually 2.7
 python: 3.5
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
This will have absolutely no effect and probably should just be removed so travis uses its default. The containers used in the build and test phases are different from this one. The actual image used is controlled by the build_wheel and install_run function calls in this file, which set the docker image to use from multibuild/travis_linux_steps.sh